Automattic-Tracks-iOS/Model/UploadError.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+public enum UploadError: Error, LocalizedError {
```

Comment:
Yep, no objection – I also thought about moving it inside EventLoggingNetworkService since it's only emitted from there, but I'm not sure if we have a style preference for this?

I went with an even-more-specific `EventLoggingFileUploadError`.

Changed in 6b135c5a6280eae2dc9a30c5be904cfc2d214e94

Reply:
I generally start with a `private enum` in these cases and only move it to a `public enum` if it'll be used elsewhere. I don't think there is any hard & fast rules about this, so whatever makes the most sense for you is fine by me.

Resolving this comment because I don't think any change _needs_ to be made, but feel free to make it, unresolve the comment and add a comment about the change :)